src/Serializers/ItemSerializer.php

```diff
@@ -102,13 +102,12 @@ private function addStatementListToSerialization( Item $item, array &$serializat
 	}
 
 	private function addSiteLinksToSerialization( Item $item, array &$serialization ) {
-		$siteLinks = $item->getSiteLinks();
-
 		$serialization['sitelinks'] = array();
 
-		foreach( $siteLinks as $siteLink ) {
+		foreach( $item->getSiteLinkList()->toArray() as $siteLink ) {
```

Comment:
@JeroenDeDauw does this to avoid the line `/** @var SiteLink $siteLink */`. Both is valid. Personally I like the `@var` solution more.